js-miniapp-bridge/src/index.ts

```diff
@@ -44,6 +46,8 @@ export {
   AccessTokenData,
   Contact,
   Points,
+  HostEnvironmentInfo,
+  Platform,
```

Comment:
Isn't `Platform` part of `HostEnvironmentInfo`?

Reply:
@Climbatize the Platform is a model itself it's only imported into HostEnvironmentInfo. not totally sure but that needs to be exported as well I think?

```
-> platform.ts
/** @internal */

/** Device platform. */
export enum Platform {
  ANDROID = 'Android',
  IOS = 'iOS',
}
```

```
import { Platform } from './platform';

/** HostEnvironmentInfo type. */
export interface HostEnvironmentInfo {
  platform?: Platform;
  platformVersion?: string;
  hostVersion?: string;
  sdkVersion?: string;
}
```